governance/polls/WBTC Risk Premium Adjustment - August 24, 2020

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Change filename to: 'Proposal - Adjust the WBTC-A Risk Premium - August 24, 2020.md'